packages/quantic/cypress/integration/standalone-search-box/standalone-search-box-expectations.ts

```diff
@@ -75,6 +75,13 @@ function standaloneSearchBoxExpectations(
         })
         .logDetail('should log the "searchFromLink" UA event');
     },
+    fetchQuerySuggestWithParams: (
+      params: Record<string, string | number | boolean>,
+      alias: string
+    ) => {
+      cy.wait(alias);
+      cy.logDetail(`should fetch query suggestions with ${params}`);
```

Comment:
You should serialize `params` if you want the content to be displayed in the log.

```suggestion
      cy.logDetail(`should fetch query suggestions with ${JSON.stringify(params)}`);
```